new-errors-in-recent-builds.rst

```diff
@@ -0,0 +1,93 @@
+.. post:: November 2, 2021
+    :tags: builders
+    :author: Anthony
+    :location: GEG
+
+.. meta::
+    :description lang=en:
+        TBD
+
+New errors in recent builds
```

Comment:
:construction:

Reply:
It's probably not perfect but I changed this to "Build errors with docutils 0.18". I think the body of the post does a great job at explaining what's happening in layman terms, but still clearly communicating the issue in the title in a non-time dependent way (i.e. avoid "new" and "recent") is necessary.